packages/f-searchbox/src/components/formElements/FormSearchInnerFieldWrapper.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+    <div :class="$style['c-search-innerFields']">
+        <form-search-geo
+            v-if="isGeoLocationAvailable"
+            :copy="copy"
+            :service="service" />
+
+        <input
+            v-if="streetNumberRequired"
+            ref="streetNumberInput"
+            :value="getStreetNumberValue"
+            :class="$style['c-search-streetInput']"
+            type="input"
+            placeholder="N°">
```

Comment:
Numero sign might be better semantically here? Or does JustEatBasis not support this?
```suggestion
            placeholder="№">
```

Reply:
I'm not sure, for now, to be safe i'll keep it the same if that's ok? @ashleynolan Do you know if this font would work ok here?